### PR TITLE
Set a max-width for full-text molecules

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
## Changes

- Set a max-width for full-text molecules

## Testing

-  Visit `/privacy/digital-privacy-policy/` if you have the DB or just create a page with a full-width-text blob.

## Review

- @anselmbradford 
- @sebworks 
- @jimmynotjim 

## Screenshots

image

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by jimmynotjim** [comment]
:+1:

File: cfgov/unprocessed/css/molecules/full-width-text.less [hunk 1]
```diff
@@ -35,6 +35,9 @@
 */
 
 .m-full-width-text {
+    // This makes for line lengths between 85-95 characters
+    max-width: 41.8em;
```
Comment (Scotchester) [inline cfgov/unprocessed/css/molecules/full-width-text.less]: 41.875? :trollface:

Comment (KimberlyMunoz) [inline cfgov/unprocessed/css/molecules/full-width-text.less]: I'm going with what's in JIRA on this.

**Comment by sebworks** [comment]
:+1: , pending @Scotchester  approval.

**Comment by Scotchester** [comment]
Hah, don't wait for me. I was just joking.

**Pull request merged**